2015/proposals/banaszkiewicz-piotr-amy.md

```diff
@@ -0,0 +1,214 @@
+# Enhance Amy, a workshop-management platform for Software Carpentry
+
+## Abstract
+
+The number of Software Carpentry's workshops run weekly dynamically grows. All
```

Comment:
I'm using [Pandas](http://pandas.pydata.org/) with [Seaborn](http://stanford.edu/~mwaskom/software/seaborn/) for nicer colors.

Pandas is very straightforward when it comes to reading and plotting CSV date-indexed files:

```python
df = pandas.read_csv("enrolment_workshops_data.csv", index_col=0, parse_dates=True)
df.plot()
```

However, if I understand correctly, Pandas has support for multi-index (multi-indices?) data frames. It would take me 5 minutes to remove duplicate data points from the CSV files, but way longer to figure out how to do that in Pandas.